TrackletAlgorithm/MatchCalculator.h

```diff
@@ -868,9 +868,9 @@ void MatchCalculator(BXType bx,
     goodmatch      = goodmatch_next;
     projseed       = projseed_next;
 
-  }// end MC_LOOP 
+    if (istep==kMaxProc-kMaxProcOffset(module::MC)-1) bx_o = bx;
```

Comment:
@bryates Looks like you undid changes here that @djcranshaw made in PR #130.

Could you please un-undo these changes? I think this file should match what's on the master branch.